django/contrib/postgres/functions.py

```diff
@@ -3,17 +3,9 @@
 
 class RandomUUID(Func):
     template = 'GEN_RANDOM_UUID()'
-
-    def __init__(self, output_field=None, **extra):
```

Comment:
I think that's OK too. It simply looks a bit odd that all that `Func` machinery is used to output constant string.